draft-ietf-tls-subcerts.md

```diff
@@ -310,8 +275,7 @@ The delegated credential has the following structure:
 
 scheme:
 
-: The signature algorithm used to sign the delegated credential, where
-  SignatureScheme is as defined in the TLS 1.3 standard.
+: The signature algorithm used to sign the delegated credential.
```

Comment:
maybe we can call this leafSigningScheme

Reply:
I'll change this in a different commit.